inji-web/src/__tests__/components/DataShare/DataShareContent.test.tsx

```diff
@@ -0,0 +1,58 @@
+import {fireEvent, render, screen} from "@testing-library/react";
+import {DataShareContent} from "../../../components/DataShare/DataShareContent";
+import {reduxStore} from "../../../redux/reduxStore";
+import {Provider} from "react-redux";
+
+describe("Test the Layout of the Expiry Content", () => {
+
+    const customMockFn = jest.fn();
+    beforeEach(() => {
+        jest.mock("react-i18next", () => ({
+            useTranslation: () => ({
+                t: (key: string) => key,
+            }),
+        }));
```

Comment:
move this to TestUtils and use in all unit tests.

Reply:
Moved to TestUtils and using it from there